addon/components/single-document-details.js

```diff
@@ -69,6 +75,11 @@ export default class SingleDocumentDetailsComponent extends Component {
 
   @action toggle(name) {
     this[name] = !this[name];
+    if (name === "editTitle" && !this[name]) {
+      // if the user cancel title editing,
+      // return edit title input value to be the orginal title
+      this.title = this.args.document.title;
+    }
```

Comment:
Let's fix it for the description as well, that way you might be able to use `[name]` generically